presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -291,6 +291,11 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
             return ImmutableSet.of();
         }
 
+        Optional<Set<String>> catalogs = filterString(constraint.getSummary(), CATALOG_COLUMN_HANDLE);
```

Comment:
How can it be wrong?

Reply:
yes it is, so when user specifies something wrong like `WHERE table_catalog = 'out_of_nowhere'` then there is no point to enumerate metadata objects.

Reply:
Please see added test